plugins/catalog-backend-module-bitbucket-server/api-report.md

```diff
@@ -0,0 +1,97 @@
+## API Report File for "@backstage/plugin-catalog-backend-module-bitbucket-server"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { BitbucketServerIntegrationConfig } from '@backstage/integration';
+import { Config } from '@backstage/config';
+import { Entity } from '@backstage/catalog-model';
+import { EntityProvider } from '@backstage/plugin-catalog-backend';
+import { EntityProviderConnection } from '@backstage/plugin-catalog-backend';
+import { LocationSpec } from '@backstage/plugin-catalog-backend';
+import { Logger } from 'winston';
+import { TaskRunner } from '@backstage/backend-tasks';
+
+// @public (undocumented)
+export type BitbucketProject = {
+  key: string;
+};
+
+// @public (undocumented)
+export type BitbucketRepository = {
+  project: {
+    key: string;
+  };
+  slug: string;
+  description: string;
+  links: Record<
+    string,
+    {
+      href: string;
+    }[]
+  >;
+};
+
+// @public
+export class BitbucketServerClient {
+  constructor(options: { config: BitbucketServerIntegrationConfig });
+  // (undocumented)
+  static fromConfig(options: {
+    config: BitbucketServerIntegrationConfig;
+  }): BitbucketServerClient;
+  // (undocumented)
+  getRepository(projectKey: string, repo: string): Promise<BitbucketRepository>;
+  // (undocumented)
+  listProjects(options?: ListOptions): Promise<PagedResponse<BitbucketProject>>;
+  // (undocumented)
+  listRepositories(
+    projectKey: string,
+    options?: ListOptions,
+  ): Promise<PagedResponse<BitbucketRepository>>;
+  // (undocumented)
+  resolvePath(project: string, repository: string, path: string): string;
```

Comment:
`resolvePath(opts: { projectKey: string, repo: string, path: string): { path: string })`

let's try to keep the names for the arguments the same in the `api-report` and also let's return objects so, like the argument input we can change the api without breaking stuff. :)